merkletree/pad.go

```diff
@@ -122,6 +122,7 @@ func (pad *PAD) GetSTR(epoch uint64) *SignedTreeRoot {
 func (pad *PAD) TB(key string, value []byte) (*TemporaryBinding, error) {
 	str := pad.latestSTR
 	index, _ := pad.computePrivateIndex(key, pad.policies.vrfPrivate())
+
```

Comment:
I think we either want to keep the VRF proof here, or return the TB with a proof of absence as I mentioned in https://github.com/coniks-sys/coniks-go/pull/29#discussion_r72140992, no?